### PR TITLE
Disabled wrong warning in GCC12

### PR DESCRIPTION
**Pull request by JohanMabille**

### CODE REVIEW

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -69,6 +69,12 @@ if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID MATCHES "GNU"
         if (NOT CROSS_COMPILE_ARM)
             set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fPIC")
         endif()
+
+        if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 12)
```
Comment (serge-sans-paille) [inline test/CMakeLists.txt:73]: greater or just equal ?

Comment (JohanMabille) [reply test/CMakeLists.txt:73]: indeed maybe equal should be enough? However, this does not seem enough to fix the CI :s

**Comment by serge-sans-paille** [comment]
Still fails for two config :-/ The warning suppression doesn't seem to be passed down on these config

**Comment by JohanMabille** [comment]
@serge-sans-paille I suggest that we remove the MINGW64 and UCRT64 builds for now since we cannot wrokaround GCC's bug. I hope they will fix it in the next GCC and that it will be available in msys2 soon.

**Pull request merged**